Apps/WebClient/src/ClientApp/src/views/timeline.vue

```diff
@@ -329,7 +329,7 @@ export default class TimelineView extends Vue {
             <b-col id="timeline" class="col-12 col-lg-9 column-wrapper">
                 <div class="px-2">
                     <b-alert
-                        :show="hasNewTermsOfService"
+                        v-show="hasNewTermsOfService"
```

Comment:
what this necessary for the unit tests?

Reply:
Yes. Not sure why, but jest was reporting that it was visible if ```:show``` was used

Reply:
This is a difference between using the show property on the BootstrapVue component and Vue's built in way to toggle "display: none" and I don't think they're equivalent.

This is what [the docs](https://bootstrap-vue.org/docs/components/alert#visible-state) say about the component:
> Use the show prop to control the visibility state of the alert. By default alerts are not shown. Set the prop show to explicitly display them.
>
> The show prop accepts boolean true or false to show and hide the alert respectively. It can also be set to a positive integer (representing seconds) to create a self dismissing alert.

It seems like this is the intended way to control visibility of the component and we should stick with that.